web/src/composebox_typeahead.js

```diff
@@ -47,6 +47,7 @@ import {user_settings} from "./user_settings";
 export const max_num_items = 8;
 
 export let emoji_collection = [];
+export let compose_typeahead_instance;
```

Comment:
This is unused, dropped this as I think it was replaced by `set_compose_textarea_typeahead` when you realized the import flow.

Reply:
Oops!, thanks for catching.